frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card-item.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FieldLevelHelp, humanizeBinaryBytes } from '@console/internal/components/utils';
+
+const ItemBody: React.FC<ItemBodyProps> = React.memo(
+  ({ title, stats, infoText, isLoading, error }) => {
+    let status: React.ReactElement;
+    if (isLoading) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error || !stats) {
+      status = <span className="co-dashboard-text--small text-muted">Not available</span>;
+    } else {
+      status = <span className="ceph-storage-efficiency-card__row-status-item-text">{stats}</span>;
+    }
+    return (
+      <div className="co-inventory-card__item">
+        <div className="ceph-storage-efficiency-card__row-title">{title}</div>
+        <div className="ceph-storage-efficiency-card__row-status-item">
+          {status}
+          <FieldLevelHelp>{infoText}</FieldLevelHelp>
+        </div>
+      </div>
+    );
+  },
+);
+
+export const SavedItem: React.FC<SavedItemProps> = React.memo(({ saved, isLoading, error }) => {
+  const infoText = '';
+  let stats: string = saved;
+  const savingsValue = Number(saved);
+  if (saved) {
+    const savedBytes = humanizeBinaryBytes(savingsValue).string;
+    stats = savingsValue <= 0 ? 'No Savings' : savedBytes;
```

Comment:
Will you ever get a negative value here ?

Reply:
Good catch, no, it can be 0 or more only